### ISSUE TITLE
Change result not found picture in dark mode

### ISSUE THREAD

**Issue by yuanLeeMidori** [Seneca-CDOT/telescope#2178]
**What happened**:
As @HyperTHD suggested, the picture of result not found in dare mode is a bit hard to see. I'd like to know what other developers think? Should we change a picture for dark mode?

![image](https://user-images.githubusercontent.com/50813726/115272218-278a8100-a10c-11eb-9eec-788150fb12d9.png)

**How to reproduce it (as precise as possible)**:
Search `jhk` in search page, you should see the result not found picture. Toggle to dark mode.

**Comment by PedroFonsecaDEV** [comment]
@izhuravlev

**Comment by izhuravlev** [comment]
I think the issue should be changed to not the picture, but the "No Results Found" message, which is just not visible because of the dark font colour. 

If we want to make the image itself visible, we can probably add a little white container behind the image.

**Comment by chrispinkney** [comment]
I personally think it's fine as is atm.
